### PR TITLE
Bug 1619253 - Document timing behavior on Android/iOS

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: docs/user/metrics/timespan.md [hunk 1]
```diff
@@ -236,7 +236,15 @@ HistorySync.setRawNanos(duration)
...
-* None.
+* Timings are recorded in nanoseconds.
+
+  * On Android, the [`SystemClock.elapsedRealtimeNanos()`](https://developer.android.com/reference/android/os/SystemClock.html#elapsedRealtimeNanos()) function is used, so it is limited by the accuracy and performance of that timer. The time measurement includes time spent in sleep.
+
+  * On iOS, the [`mach_absolute_time`](https://developer.apple.com/library/archive/documentation/Darwin/Conceptual/KernelProgramming/services/services.html) function is used,
+    so it is limited by the accuracy and performance of that timer.
+    The time measurement does not include time spent in sleep.
+
+  * On Python 3.7 and later, [`time.monotonic_ns()`](https://docs.python.org/3/library/time.html#time.monotonic_ns) is used.  On earlier versions of Python, [`time.monotonics()`](https://docs.python.org/3/library/time.html#time.monotonic) is used, which is not guaranteed to have nanosecond resolution.
```
Comment (Dexterp37) [inline docs/user/metrics/timespan.md:247]: @badboy does this include sleep time?

Comment (badboy) [reply docs/user/metrics/timespan.md:247]: Probably depends on the OS :/

Comment (Dexterp37) [reply docs/user/metrics/timespan.md:247]: Ok, worth calling that out.

Comment (badboy) [reply docs/user/metrics/timespan.md:247]: fair point, will followup

**Pull request merged**